.pre-commit-config.yaml

```diff
@@ -1,8 +1,4 @@
-exclude: |
-    (?x)^(
-        .*/vendor/.*|
-        .github/CODEOWNERS
-    )$
```

Comment:
Could you explain why this needs to not be excluded?

Reply:
The whole point of this PR was to fix the trailing white space in `CODEOWNERS`, so that we would only exclude true 3rd party code (nothing but the unittest library). I didn't know that trailing whitespace in `CODEOWNERS` was a feature. If that is the case, we should regard `CODEOWNERS` as something external from the perspective of this repo as well and leave everything as-is. Sorry for the noise then.

Reply:
But why would you want to remove the trailing newline? https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Ah, now I understand!